java/HealthVault Mobile/HVSample/src/main/res/layout/weight.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I have created task # 54619 for this comment

---
In reply to: [112366852](https://github.com/Microsoft/healthvault-samples/pull/4#discussion_r112366852) [](ancestors = 112366852)